cloudinit/config/cc_install_hotplug.py

```diff
@@ -129,8 +129,12 @@ def handle(_name, cfg, cloud, log, _args):
         log.debug("Skipping hotplug install, udevadm not found")
         return
 
+    # This may need to turn into a distro property at some point
```

Comment:
+1 I'm good with this approach. Checking /usr/libexec dir first would allow us to prefer that on distros that have installed cloud-init to that path.